blog.html

```diff
@@ -31,6 +31,7 @@ <h2 class="bullet">
                 <p>We are proud to announce the release of Apache Kafka 3.5.0. This release contains many new features and improvements. This blog post will highlight some of the more prominent features. For a full list of changes, be sure to check the <a href="https://downloads.apache.org/kafka/3.5.0/RELEASE_NOTES.html">release notes</a>.</p>
                 <p>See the <a href="https://kafka.apache.org/35/documentation.html#upgrade_3_5_0">Upgrading to 3.5.0 from any version 0.8.x through 3.4.x</a> section in the documentation for the list of notable changes and detailed upgrade steps.</p>
                 <p>The ability to migrate Kafka clusters from ZK to KRaft mode with no downtime is still an early access feature. It is currently only suitable for testing in non production environments. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-866+ZooKeeper+to+KRaft+Migration">KIP-866</a> for more details.</p>
+		<p><it>Note: ZooKeeper is now marked deprecated in this release. ZooKeeper is planned to be removed in Apache Kafka 4.0. (Cf <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-833%3A+Mark+KRaft+as+Production+Ready#KIP833:MarkKRaftasProductionReady-Kafka3.5">KIP-833</a>)</it><p>
```

Comment:
I would probably make it more clear over there what deprecated means. We will continue to support and backport bug fixes for Zk in 3.x releases. With 4.x release, Zk will be removed from Apache Kafka.

A natural next question in the mind of reader comes as when is 4.0 happening. We should also mention that release is to-be-decided.

Reply:
IMO, it's not great to add this important note after the 3.5.0 announcement published.

Reply:
We should have added this before and it was a miss. But it's better to add it now than not at all. The key reason for doing this is to give as much advance notice as possible. The KIP linked to does have a target date for 4.0, but it's subject to change (as usual).